plugins/stateful/src/apollo.ts

```diff
@@ -0,0 +1,73 @@
+import { ApolloClient, ApolloLink, FetchResult, HttpLink, InMemoryCache, Operation, split } from '@apollo/client';
+import { setContext } from '@apollo/client/link/context';
+import { getMainDefinition } from '@apollo/client/utilities';
+import { Client, ClientOptions, createClient } from "graphql-sse";
+
+import {
+  Observable
+} from '@apollo/client/core';
+import { print } from 'graphql';
+
+class SSELink extends ApolloLink {
+  private client: Client;
+ 
+  constructor(options: ClientOptions) {
+    super();
+    this.client = createClient(options);
+  }
+ 
+  public request(operation: Operation): Observable<FetchResult> {
+    return new Observable((sink) => {
+      return this.client.subscribe<FetchResult>(
+        { ...operation, query: print(operation.query) },
+        {
+          next: sink.next.bind(sink),
+          complete: sink.complete.bind(sink),
+          error: sink.error.bind(sink),
+        },
+      );
+    });
+  }
+}
```

Comment:
@duvanmonsa I thought SSE should work by default with @apollo/client, similar as how it does with RedwoodJS. I guess `graphql-sse` is needed because it doesn't work, but why?

Reply:
SSE is required!